bot/exts/utilities/bookmark.py

```diff
@@ -71,6 +105,22 @@ class Bookmark(commands.Cog):
 
     def __init__(self, bot: Bot):
         self.bot = bot
+        self.book_mark_context_menu = discord.app_commands.ContextMenu(
+            name="Bookmark",
+            callback=self._bookmark_context_menu_callback
+        )
+        self.bot.tree.add_command(self.book_mark_context_menu)
+
+    @staticmethod
+    def build_bookmark_embed(target_message: discord.Message) -> discord.Embed:
+        """Build the channel embed to the bookmark requester."""
+        return discord.Embed(
+            description=(
+                f"Click the button to be sent your very own bookmark to "
+                f"[this message]({target_message.jump_url})."
```

Comment:
See my comment above. I don't think this description is necessary either. We just need something like "Bookmark sent! Check your DMs" or something similar

Reply:
Makes sense, done in cf4903561a222de9b7413c406b5d6e186f977332